pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-common/src/main/java/org/apache/pinot/plugin/ingestion/batch/common/SegmentGenerationTaskRunner.java

```diff
@@ -60,7 +60,9 @@
   public static final String EXCLUDE_SEQUENCE_ID = "exclude.sequence.id";
 
   // Assign sequence ids to input files based at each local directory level
-  public static final String LOCAL_DIRECTORY_SEQUENCE_ID = "local.directory.sequence.id";
+  @Deprecated
+  public static final String DEPRECATED_USE_LOCAL_DIRECTORY_SEQUENCE_ID = "local.directory.sequence.id";
+  public static final String USE_LOCAL_DIRECTORY_SEQUENCE_ID = "use.local.directory.sequence.id";
```

Comment:
If we use local directory sequence id by default, then I would suggest configuring global sequence id so that the default is false.

Reply:
so you mean to change this config to `use.global.directory.sequence.id`?